udata_search_service/presentation/api.py

```diff
@@ -93,6 +93,27 @@ def sort_validate(cls, value):
         return value
 
 
+class DataserviceArgs(BaseModel):
+    q: Optional[str] = None
+    page: Optional[int] = 1
+    page_size: Optional[int] = 20
+    sort: Optional[str] = None
+    tag: Optional[str] = None
+    organization: Optional[str] = None
+    owner: Optional[str] = None
+    is_restricted: Optional[bool] = None
+
+    @validator('sort')
+    def sort_validate(cls, value):
+        sorts = [
+            'created',
+        ]
+        choices = sorts + ['-' + k for k in sorts]
+        if value not in choices:
+            raise ValueError('Sort parameter is not in the sorts available choices.')
+        return value
```

Comment:
Nit: is this for future sorts? It feels a bit overkill for the moment 🤣

Reply:
[sorts](https://github.com/opendatateam/udata-search-service/pull/48/files/82debfdfe1ab2711a586805c75ae901829cd3816#diff-c82642483ae4c76bf0685cb300bbea5848e8a4f8a8cc710358d52843756d2406R108-R110) are just one element, `created`, right? So this whole code could be shortened to

```python
sorts = ["created", "-created"]
if value not in sorts:
...
```

No big deal, just looked a bit funny ;)

Reply:
Indeed, it's the same logic in other models, Datasets having more sorts for example